code/modules/mod/mod_theme.dm

```diff
@@ -1159,6 +1159,88 @@
 	acid = 100
 	wound = 25
 
+/datum/mod_theme/interdyne
+	name = "interdyne"
+	desc = "A corpse-snatching and rapid-retrieval modsuit, resulting from a lucrative tech exchange between Interdyne Pharmaceutics and Cybersun Industries."
+	extended_desc = "While Waffle Co. and Azik Interstellar provide the means, Donk Co., Tiger Cooperative, Animal Rights Consortium and \
+		Gorlex Marauders willing or easly bribable brawn, S.E.L.F. and MI13 information, the clear syndicate tech providers would be Interdyne and Cybersun, \
+		their combined knowledge in technologies rivaled by only most enigmatic of aliens, and certainly not by any Nanotrasen scientist. \
+		This model is one of rare fruits created by their joint operations, mashing scrapped designs with super soldier enhancements. \
+		Already light, when powered on, this modsuits injects the wearer seemlessly with muscle-enhancing supplements, while adding piston strenght \
+		to their legs. Combination of these mechanisms is very energy draining - but results in next to no speed reduction for the wearer.\
+		Over the years, many a rich person, including Nanostrasen officials with premium subscriptions, had their life or genes rescued thanks to \
+		unrivaled speed of this suit model. Equally as many, however, mysteriously dissapeared in the flash of these white suits after they forgot \
+		to pay off said subscriptions in due time or publicly communicated unfavourable opinions on Intrudyne's gene-modding tech and ethics. "
+	default_skin = "interdyne"
+	armor_type = /datum/armor/mod_theme_interdyne
+	resistance_flags = FIRE_PROOF|ACID_PROOF
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	charge_drain = DEFAULT_CHARGE_DRAIN * 2
+	slowdown_inactive = 0.0
+	slowdown_active = -0.5
+	inbuilt_modules = list(/obj/item/mod/module/quick_carry/advanced, /obj/item/mod/module/organ_thrower)
+	allowed_suit_storage = list(
```

Comment:
can you alphabetize this list? there's extensions for vscode that will do it automatically for you with a simple keybind. just helps maintainability a lot